packages/server/tests/acceptance/rpc_batch2.spec.ts

```diff
@@ -70,6 +70,14 @@ describe('@api-batch-2 RPC Server Acceptance Tests', function () {
     let blockNumberAtStartOfTests = 0;
     let mirrorAccount0AtStartOfTests;
 
+    const signAndSendTransaction = async (transaction, accounts, requestId) => {
```

Comment:
nit: naming update suggestion
```suggestion
    const signSendAndConfirmTransaction = async (transaction, accounts, requestId) => {
```